### PR TITLE
Batch update exchange

### PR DESCRIPTION
**Pull request by EldinZhou**
Add batch update statements for vertices and edges with unit tests.

### CODE REVIEW

File: exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala [hunk 1]
```diff
@@ -248,10 +345,10 @@ class NebulaGraphClientWriter(dataBaseConfigEntry: DataBaseConfigEntry,
   }
 
   override def writeEdges(edges: Edges, ignoreIndex: Boolean = false): String = {
-    val sentence = toExecuteSentence(config.name, edges, ignoreIndex)
+    // val sentence = toExecuteSentence(config.name, edges, ignoreIndex)
```
Comment (Nicole00) [inline exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala:348]: you can just remove this line. same for writeVertices.

**Comment by Nicole00** [comment]
LGTM！great job, thanks for your contribution~ @EldinZhou

File: exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala (hunk 1 above)
Comment (EldinZhou) [reply exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala:348]: OK, I got it~

Comment (EldinZhou) [reply exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala]: > you can just remove this line. same for writeVertices.

Done~

**Review by Nicole00 — APPROVED** [review]

**Comment by Nicole00** [comment]
@cooper-lzy Now we can support batch update function in Exchange, the config is the same as batch delete: there's one extra config named `writeMode` for each tag or edge. the value can be `insert`,`delete` and `update`, and default value is `insert`.  https://github.com/vesoft-inc/nebula-exchange/pull/151/files#diff-a55ebb9daa91e9046a352946a76fff36b82f42d2a31881983c4a57abb2b70f8d

**Pull request merged**